app/livechat/server/lib/Helper.js

```diff
@@ -275,18 +275,32 @@ export const forwardRoomToDepartment = async (room, guest, transferData) => {
 	}
 	callbacks.run('livechat.beforeForwardRoomToDepartment', { room, transferData });
 	const { _id: rid, servedBy: oldServedBy, departmentId: oldDepartmentId } = room;
+	let agent = null;
 
 	const inquiry = LivechatInquiry.findOneByRoomId(rid);
 	if (!inquiry) {
 		throw new Meteor.Error('error-transferring-inquiry');
 	}
 
 	const { departmentId } = transferData;
-
 	if (oldDepartmentId === departmentId) {
 		throw new Meteor.Error('error-forwarding-chat-same-department', 'The selected department and the current room department are the same', { function: 'forwardRoomToDepartment' });
 	}
 
+	const { userId: agentId } = transferData;
+	if (agentId) {
+		let user = Users.findOneOnlineAgentById(agentId);
+		if (!user) {
+			throw new Meteor.Error('error-user-is-offline', 'User is offline', { function: 'forwardRoomToAgent' });
+		}
+		user = LivechatDepartmentAgents.findOneByAgentIdAndDepartmentId(agentId, departmentId);
+		if (!user) {
+			throw new Meteor.Error('error-user-not-belong-to-department', 'The selected user does not belong to this department', { function: 'forwardRoomToDepartment' });
```

Comment:
The translations are missing for this error: `error-user-not-belong-to-department`.
At least `pt-BR` and `EN` must be defined.